### DISCUSSION TITLE
Works without javascript?!

### DISCUSSION THREAD

**Discussion by hinogi** [remix-run/remix#3122] [Q&A] [answered]
I'm a bit confused, that is both javascript?!
HTML forms work with no extra javascript but here javascript is necessary to generate a html form, i'm confused.
![image](https://user-images.githubusercontent.com/4602609/167249434-299a1da0-59cc-4939-a96d-befc96451daf.png)

**Comment by ijohn** [comment, marked as answer]
The "works with no JavaScript!" part means it works even when the browser has JavaScript disabled.
